### ISSUE TITLE
Updation of EmbeddedPictures Description Bug?

### ISSUE THREAD

**Issue by DividedSE** [Zeugma440/atldotnet#114]
Well found another thing...
Updating the description only supposed to not work!!!

```
track.EmbeddedPictures[index].Description= "New Description"
track.save();
```
also

```
data.description="test";
track.embededdpictures.removeat(index);
track.embeddedpictures.insert(index,data);
track.save();
```
Not working
is it how supposed to work? Or not?

**Comment by Zeugma440** [comment]
These are supposed to work, yes.

Which players are you using to read picture descriptions ?

Right now it looks like there's a slight formatting issue - it might work on some players and fail on some others until it is fixed.

**Comment by Zeugma440** [comment]
Please let me know if it works for you now.

If it doesn't, please tell me which player you are using to visualize picture description.

**Comment by DividedSE** [comment]
Not working.. when i add a new picture with description it works ( before & after the fix)..
But if a just add the picture save it, add a description separately save .. it doesnt save it.

**Comment by DividedSE** [comment]
```
Var data=track.EmbeddedPictures.FirstorDefault(x=>x.Nativepiccode == 05)
If(data == null) return;
Var index = track.embeddedpicture.indexof(data);
data.description="demo value";
track.embedededpictures.removeat(index);
track.save(); // doesn't work without this save. If i remove this it will not update.
track.embedespictures.insert(index,data);
track.save();
```
So i need to double save to update the description which is not ideal. so can you check again?

**Comment by Zeugma440** [comment]
Sorry about that, I focused on the formatting issue I found and didn't realize your issue was related to updating an existing picture.

That should be fixed now.

**Comment by Zeugma440** [comment]
The fix has been published on today's v3.25

**Comment by Zeugma440** [comment]
No feedback from the OP since v3.25 release => Closed

Please reopen the issue if the problem persists.

Please open a new issue if a different problem arises.
